src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -1574,6 +1644,163 @@ def get_lambda(real_lut, imag_lut=None):
             interp_arrays.append(interp_coupling_matrix)
         return tuple(interp_arrays)
 
+    def _interp_az_za_map_coordinates(
+        self,
+        *,
+        az_array,
+        za_array,
+        freq_array,
+        freq_interp_kind="cubic",
+        freq_interp_tol=1.0,
+        polarizations=None,
+        spline_opts=None,
+        check_azza_domain: bool = True,
+        reuse_spline: bool = False,
+    ):
+        """
+        Interpolate in az_za coordinate system using RectBivariateSpline.
```

Comment:
I thought this one didn't use RectBivariateSpline?

Reply:
Oops, typo there. Thanks for catching that. That's been updated now.